api/views/student.py

```diff
@@ -17,10 +17,19 @@
     StudentWriteSerializer,
 )
 from api.serializers.errors import BaseAPIExceptionSerializer, ValidationErrorSerializer
-from api.views.mixins import ReadWriteSerializersMixin
+from api.views.mixins import (
+    ReadWriteSerializersMixin,
+    StudentReturnedFromLeaveMixin,
+    StudentWentOnLeaveMixin,
+)
 
 
-class StudentViewSet(ReadWriteSerializersMixin, viewsets.ModelViewSet[Student]):  # type: ignore
+class StudentViewSet(  # type: ignore
+    ReadWriteSerializersMixin,
+    viewsets.ModelViewSet[Student],
+    StudentWentOnLeaveMixin,
+    StudentReturnedFromLeaveMixin,
+):
```

Comment:
Hm... Why is it so that after so many actions we have already added (e.g. transfer from group to group etc.) this is the first time we're adding some mixins to the viewsets?

Reply:
Because `Coordinator or bot` were only in the last two issues :)